rpc/wrtc_signaling.go

```diff
@@ -44,8 +54,21 @@ func extendWebRTCConfig(original *webrtc.Configuration, optional *webrtcpb.WebRT
 		iceServers := make([]webrtc.ICEServer, len(original.ICEServers)+len(optional.AdditionalIceServers))
 		copy(iceServers, original.ICEServers)
 		for _, server := range optional.AdditionalIceServers {
+			urls := server.Urls
+			if replaceUDPWithTCP {
+				urls = nil
+				for _, url := range server.Urls {
+					if strings.HasSuffix(url, "udp") {
```

Comment:
I tried to remove this functionality and creating a TURN client resulted in "connection refused". No TURN candidate was gathered for the rock4, the SDK (obviously) didn't receive a remote TURN candidate and no connection could be established. Any other reason to think we _don't_ need this, @edaniels ?